src/components/GoogleMapContainer.ts

```diff
@@ -26,7 +29,9 @@ interface GoogleMapContainerProps extends WrapperProps {
     addressAttribute: string;
     latitudeAttribute: string;
     longitudeAttribute: string;
+    markerImageAttribute: string;
     staticLocations: StaticLocation[];
+    markerImages: Array<{ enumKey: string, enumImage: string}>;
```

Comment:
Add in `validateProps` a check if the enumKey is available in the meta data
https://apidocs.mendix.com/7/client/mendix_lib_MxMetaObject.html#getEnumMap